workers/unity/Packages/io.improbable.gdk.buildsystem/.codegen/Source/WorkerGenerationJob.cs

```diff
@@ -18,8 +18,8 @@ public class WorkerGenerationJob : CodegenJob
         private readonly string relativeOutputPath = Path.Combine("improbable", "buildsystem");
         private readonly string relativeEditorPath = Path.Combine("improbable", "buildsystem", "Editor");
 
-        public WorkerGenerationJob(string baseOutputDir, IFileSystem fileSystem, DetailsStore detailsStore, bool force)
-            : base(baseOutputDir, fileSystem, detailsStore, force)
+        public WorkerGenerationJob(CodegenJobOptions options, IFileSystem fileSystem, DetailsStore detailsStore)
+            : base(options.AsEditor(), fileSystem, detailsStore)
```

Comment:
I quite like this way of using `.AsEditor` to signify it's for editor code

Reply:
It was either that or an attribute, and I didn't feel like writing more reflection code 😂